app/Http/Livewire/ShopComponent.php

```diff
@@ -14,11 +15,19 @@ class ShopComponent extends Component
 
     public function mount()
     {
-        $this->sorting = "default";
-        $this->pagesize = 12;
+        $this->sorting = config('constant.default_sorting');
+        $this->pagesize = config('constant.default_pagesize');
+    }
+
+    public function store($product_id, $product_name, $product_price)
+    {
+        Cart::add($product_id, $product_name, config('constant.defautl_add_to_cart_amount'), $product_price)
+            ->associate('App\Models\Product');
```

Comment:
use Product ở trên cùng nhé

Reply:
mình dùng use Product trên cùng sẽ báo lỗi này **_Use of undefined constant Product - assumed 'Product'_**

Reply:
17 files change là do mình sửa lại thêm hằng số vào config

Reply:
bn chuyền vào Product::class chứ